plugins/VisitTime/lang/en.json

```diff
@@ -5,32 +5,49 @@
         "ColumnLocalMinute": "Local time - minute (Start of visit)",
         "ColumnServerTime": "Server time",
         "ColumnServerHour": "Server time - hour",
+        "ColumnSiteHour": "Site time - hour",
         "ColumnVisitEndServerHour": "Server time - hour (Time of last action)",
+        "ColumnVisitEndSiteHour": "Site time - hour (Time of last action)",
         "ColumnVisitEndServerMinute": "Server time - minute (Time of last action)",
+        "ColumnVisitEndSiteMinute": "Time in UTC - minute (Time of last action)",
         "ColumnVisitStartServerHour": "Server time - hour (Start of visit)",
+        "ColumnVisitStartSiteHour": "Site time - hour (Start of visit)",
         "ColumnVisitStartServerMinute": "Server time - minute (Start of visit)",
+        "ColumnVisitStartSiteMinute": "Time in UTC - minute (Start of visit)",
         "ColumnVisitEndServerDate": "Server time - date (Time of last action)",
+        "ColumnVisitEndSiteDate": "Time in UTC - date (Time of last action)",
         "ColumnVisitEndServerDayOfMonth": "Server time - day of month (Time of last action)",
+        "ColumnVisitEndSiteDayOfMonth": "Time in UTC - day of month (Time of last action)",
         "ColumnVisitEndServerDayOfWeek": "Server time - day of week (Time of last action)",
+        "ColumnVisitEndSiteDayOfWeek": "Time in UTC - day of week (Time of last action)",
         "ColumnVisitEndServerDayOfYear": "Server time - day of year (Time of last action)",
+        "ColumnVisitEndSiteDayOfYear": "Time in UTC - day of year (Time of last action)",
         "ColumnVisitEndServerQuarter": "Server time - quarter (Time of last action)",
+        "ColumnVisitEndSiteQuarter": "Time in UTC - quarter (Time of last action)",
```

Comment:
It's actually a bit confusing that all translations keys are named `Site`, even though most of them actually don't contain a site related time, but UTC. Maybe renaming the keys where UTC is used would make it a bit less confusing when looking through the changes.

Reply:
@sgiehl updated